src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -183,7 +183,7 @@ public void authorConfig_emptyLocation_success() throws ParseException, IOExcept
     }
 
     @Test (expected = IOException.class)
-    public void authorConfig_emptyConfig_fail() throws IOException {
+    public void authorConfig_emptyConfig_throwsIoException() throws IOException {
```

Comment:
for this, can just use the same case letters as the `IOException` to avoid complicating the standard.

Reply:
But our checkstyle rules prohibit having more than 2 consecutive capital letters right?
I get a checkstyle error when I set the test case name to `authorConfig_emptyConfig_throwsIOException`

Reply:
I just update the test case to use the name `authorConfig_emptyConfig_throwsIOException`.

Below is the checkstyle error seen on the Travis logs:
![image](https://user-images.githubusercontent.com/35621759/69971950-ae60ad00-155b-11ea-904f-ee7361e2f651.png)

It looks like we cannot use more than 2 consecutive capital letters. Am I doing something wrong here?

Reply:
I see.  My concern was to retain original name to avoid double standard. But since its prohibited by checkstyle then I guess we can stick to `throwsIoException`.

Reply:
Done. I have changed the name back to `authorConfig_emptyConfig_throwsIoException`